script.js

```diff
@@ -1,4 +1,349 @@
+  playerArray.length = 0;
+  resultArray.length = 0;
+  highScoreArray.length = 0;
+
+  return RESET_MSG;
+};
+
+/***
+ * Play a roll of the game, rolls dice according to how much dice are in the game
+ * @returns {Array} an array of all the rolls of the player
+ */
+var rollDice = function () {
+  var diceRolls = [];
+  for (i = 0; i < diceNum; i += 1) {
+    diceRolls.push(randomizeDiceRoll());
+  }
+  return diceRolls;
+};
+
+/***
+ * displays all rolls the player has made and format it
+ * @param {Array} diceRollArray the dice rolled for the player
+ * @returns {string} player roll display
+ *
+ */
+var scoreRolls = function (diceRollArray) {
+  var rollDisplay = "";
+  for (var i = 0; i < diceRollArray.length; i += 1) {
+    rollDisplay += `Dice ${i + 1}: ${diceRollArray[i]}<br>`;
+  }
+
+  //sorting the array, normal string sort can be applied because values are only single digit
+  //if playing with a dice of more than 2 digits, need to update this code with a compare function
+  //this will sort smallest to biggest number
+  diceRollArray.sort();
+  //reverses the array if it's playing the highest mode, hence biggest to smallest
+  if (diceMode == HIGHEST_MODE) diceRollArray.reverse();
+
+  var playerScore = "";
+
+  //string variable does not add the numbers, it concactenates the string
+  for (var i = 0; i < diceRollArray.length; i += 1) {
+    playerScore += String(diceRollArray[i]);
+  }
+
+  //assigns the current player's score to the resultArray, storing them as numbers
+  currentPlayerIndex = playerArray.indexOf(currentPlayer);
+  resultArray[currentPlayerIndex] = Number(playerScore);
+
+  rollDisplay += `<br> The ${diceMode} possible score is ${playerScore}`;
+
+  return rollDisplay;
+};
+
+/***
+ * Plays standard game and it goes on every single time the function is triggered
+ * @returns {string} Result of the current roll for output
+ */
+var playStandardGame = function () {
+  var outputMsg = `Player ${currentPlayer} has thrown the dice! <br><br>`;
+  var currentPlayerRoll = rollDice();
+
+  outputMsg += scoreRolls(currentPlayerRoll);
+
+  //initialize next Player's index
+  var nextPlayerIndex = playerArray.indexOf(currentPlayer) + 1;
+
+  //if next player's index is still within the array range, assign the next player as the current player
+  if (nextPlayerIndex < playerArray.length) {
+    currentPlayer = playerArray[nextPlayerIndex];
+    outputMsg += `<br><br> It's Player ${currentPlayer}'s turn now!`;
+    return outputMsg;
+  }
+
+  //if it's not within the array range, means all player has taken a turn, output will reflect who win
+  var winningPlayerScore = Math.max(...resultArray);
+  if (diceMode == LOWEST_MODE) winningPlayerScore = Math.min(...resultArray);
+
+  var winningPlayerIndex = resultArray.indexOf(winningPlayerScore);
+
+  outputMsg += `<br><br>With a score of ${resultArray[winningPlayerIndex]}, Player ${playerArray[winningPlayerIndex]} Wins this round!<br><br>`;
```

Comment:
i think this is ok but what about the case of ties? 

e.g. if player 1 and player 3 both have the max score then this will assign the winner to be player 1